packages/mwp-app-render/src/components/connectedWithMatchMedia.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector, createSelector } from 'reselect';
+import withMatchMedia from 'meetup-web-components/lib/utils/components/withMatchMedia';
+
+/**
+ * getConfig
+ * @param  {Object} state
+ * @return {Object} the config from state
+ */
+const getConfig = state => state.config || {};
+
+/**
+ * Checks if object is empty
+ * @param  {Object} object
+ * @return {Boolean} whether object is empty
+ */
+const isEmpty = object => object === undefined || Object.keys(object).length === 0;
+
+/**
+ * @param {Object} device - object containing device information
+ * @returns {Object} - default media object for `withMatchMedia`
+ */
+const getMediaOrDefault = (media, device) => {
+	if (!isEmpty(media)) {
+		return media;
+	}
+	if ( !isEmpty(device) ) {
+		const {isMobile, isTablet, isDesktop} = device;
+		const isTabletOrDesktop = isTablet || isDesktop;
+		return {
+			isAtSmallUp: Boolean(isMobile || isTabletOrDesktop),
+			isAtMediumUp: Boolean(isTabletOrDesktop),
+			isAtLargeUp: Boolean(isDesktop),
+		}
+	}
+	return {};
+};
+
+const mapStateToProps = createStructuredSelector({
+	device: createSelector(getConfig, config => (config.device || {})),
+});
```

Comment:
the reselect-enhanced selector is overkill IMO - we can avoid the dependency and the complexity by just implementing this as

```js
const mapStateToProps = state => ({ device: state.config.device });
```

(although I'm recommending `const mapStateToProps = state => ({ media: state.config.media });` in this review!)